mustermann/lib/mustermann/ast/translator.rb

```diff
@@ -116,9 +116,19 @@ def decorator_for(node)
         result
       end
 
+      # @return [URI::RFC2396_Parser] URI::RFC2396 parser
+      # @!visibility private
+      def uri_parser
+        if defined?(URI::RFC2396_PARSER)
+          URI::RFC2396_PARSER
+        else
+          @uri_parser ||= URI::RFC2396_Parser.new
+        end
+      end
```

Comment:
CI require 100% coverage so we need to cover this brancing in the specs

Reply:
If it is possible in some resonable way? I don't know. If not, can we opt-out from the coverage requirement for this code?

Reply:
@dentarg when defined as a constant, you get 100% coverage. You can see my PR.
https://github.com/sinatra/mustermann/pull/142

Reply:
Although, I don't know where `URI::RFC2396_PARSER` comes from :(

Reply:
Oh got it. It's in the next [version](https://github.com/ruby/ruby/blob/eb144ef91e0b7f01cbe915bac2ac29bd532e1c28/lib/uri/common.rb#L16) of Ruby.